api/v1/routes/contact_us.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel, EmailStr, constr
+from fastapi.responses import JSONResponse
+import requests
+
+
+router = APIRouter()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
```

Comment:
setup the logger properly. Create a logging folder, set the name of the log files, set the format.

Also the logging level doesn't mean only `INFO` would be logged, just incase you don't know.

Reply:
I have enhanced the code to make sure it creates a log file within a directory and improved the logging system to log more details.